lively.ide/text/code-evaluation-commands.js

```diff
@@ -54,7 +54,16 @@ export const codeEvaluationCommands = [
       await cmd.whenDone();
       let result = cmd.output;
 
-      $world.execCommand('open workspace', { title: 'Blame Info', content: result, language: 'text' });
+      const text = new Text({
+        textString: result,
+        fixedWidth: true,
+        fixedHeight: true,
+        extent: pt(415, 300),
+        name: 'Blame Info',
+        fill: Color.white,
+        clipMode: 'auto'
+      });
+      text.openInWindow();
```

Comment:
This is pretty similar to https://github.dev/LivelyKernel/lively.next/blob/18e96a708be3ec51f5064818a5dbcdd84dabaccb/lively.ide/js/browser/ui.cp.js#L312. Can we maybe solve this by a shared component located somewhere in `lively.ide`?